src/main.cpp

```diff
@@ -205,6 +205,9 @@ main(int argc, char *argv[])
 
         parser.process(app);
 
+        // make sure that size_t properties will work
+        qRegisterMetaType<size_t>("size_t");
```

Comment:
I don't think they will ever work correctly, you should probably use an int instead. (A size_t will not fit into a big JS number anyway)

Reply:
I think this is just leftover anyway. I'll delete it